packages/frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "react-redux": "^9.1.0",
     "react-router-dom": "^6.22.0",
     "sass": "^1.70.0",
+    "telegram-login-button": "^0.1.0",
```

Comment:
Let's discuss this? I don't like the idea adding 
1) not released library (version 0.X.X) 
2) not fast supported library - it has open issue about missing last_name, and 2 PRs for dependecies update and owner doesn't seems to eager to fix this quick
3) Library that contains internal dependencies that we are not going to use and that adds 95Kb (unpacked) just to check fields type in runtime - https://www.npmjs.com/package/prop-types
4) library for one small file https://github.com/thisisamir98/telegram-login-button/blob/master/src/index.tsx

I suggest removing this dependecy and copy content of this file to our repo, this way we will have full control of this code and we can skip part with runtime verification, and also add last_name - we need it - to data without needing to wait until library owner fix this issue.